### DISCUSSION TITLE
Contribute - Rewrite and restructure

### DISCUSSION THREAD

**Discussion by toby63** [mumble-voip/mumble-www#120] [Ideas]
1. I think the site is very long, so it might be worth looking for ways to make it shorter:

- Use Collapsible sections (is that possible with hugo?)
- Use a Table of Content
- Add the following categories (restructure):
     - **Report bugs and suggest Improvements/Features**: combines Feature Requests and Issues (could also mention Testing)
     - **Development**:
         - Coding: includes mumble, skinning, 3rd party applications etc.
          - Non-Coding: Translations and **Testing** (could also mention Documentation)
     - **Support**: includes Documentation, Community Channels
     - **Hosting**: Host your own Mumble Server etc.
     - **Promotion**: Promote Mumble to your friends and colleagues

2. Also add some more information:
- Coding: Add commit guidelines; add important tasks that should be dealt with (gRPC development etc.)
- Documentation: Add a list of what things should be created or converted from the wiki
- rename "Skinning"

**Comment by Krzmbrzl** [comment]
I agree that this is probably a good idea. I am against including things like our commit guidelines though as this would only mean duplication of information. We should simply link to the guidelines instead.

**Reply by toby63** [reply]
I think the actual text could be included later, just like the rest of the documentation. 
This could work via converting it automatically, this way it is also synced.

In the meantime I agree, we could just add a link for now.

**Comment by Kissaki** [comment]
I will be on holiday starting from the end of the week and hope I will do some stuff for Mumble, including some more work/improvements to the website.

**Comment by toby63** [comment]
I added a draft pull request: https://github.com/mumble-voip/mumble-www/pull/123

**Comment by toby63** [comment]
This can be closed now I guess. Everything is implemented.
